matlab/Ice/greeter/README.md

```diff
@@ -17,7 +17,7 @@ cd matlab/Ice/greeter
 
 ```shell
 mkdir generated
-slice2matlab '--output-dir generated Greeter.ice'
+slice2matlab --output-dir generated Greeter.ice
```

Comment:
unrelated change, slice2matlab was fixed to accept multiple args in https://github.com/zeroc-ice/ice/pull/3527